language/move-prover/bytecode-to-boogie/src/dataflow_analysis.rs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+//! Adapted from AbstractInterpreter for Bytecode, this module defines the data-flow analysis
+//! framework for stackless bytecode.
+
+use bytecode_verifier::absint::{AbstractDomain, JoinResult};
+use bytecode_verifier::control_flow_graph::{BlockId, ControlFlowGraph};
+use stackless_bytecode_generator::stackless_bytecode::StacklessBytecode;
+use std::collections::HashMap;
+
+pub struct BlockState<State: Clone> {
+    pub pre: State,
+    pub post: State,
+}
+
+pub type StateMap<State> = HashMap<BlockId, BlockState<State>>;
+
+/// Take a pre-state + instruction and mutate it to produce a post-state。
+pub trait TransferFunctions {
+    type State: AbstractDomain + Clone;
+
+    /// Execute local@instr found at index local@index in the current basic block from pre-state
```

Comment:
Is local@instr Rust documentation markup? Never seen it (but not yet familiar with their markups).

If it is your personal convention, I would recommend instead using `index`. This is typically what is used in markup, and understood by most documentation generators to map agains the declared parameters.